### PR TITLE
2020.1.3 tls components support

### PR DESCRIPTION
**Pull request by NormJohnIV**
- add support for components
- add support for 2020.1.3+ tls overlays

### CODE REVIEW

File: roles/vdm/library/siteconfig_info.py [hunk 1]
```diff
@@ -2,38 +2,28 @@
 import glob
 import yaml
 import os
+from enum import Enum, auto
+
+class Overlay(Enum):
+  COMPONENT = "components"
```
Comment (thpang) [inline roles/vdm/library/siteconfig_info.py:8]: Just curious why are the defined items singular when the assigned values are plural?

File: roles/vdm/library/siteconfig_info.py [hunk 2]
```diff
@@ -2,38 +2,28 @@
...
+  CONFIGURATION = "configurations"
+  GENERATOR = "generators"
+  RESOURCE = "resources"
+  TRANSFORMER = "transformers"
```
Comment (thpang) [inline roles/vdm/library/siteconfig_info.py:12]: Just curious why are the defined items singular when the assigned values are plural?

**Comment by hahewlet** [comment]
Testing is completed. Signing off.

**Comment by agianni** [comment]
This should say 2021.1.3 at the top right?

**Pull request merged**